code/game/objects/structures/crates_lockers/closets/malfunction.dm

```diff
@@ -9,8 +9,7 @@
 	..()
 	new /obj/item/weapon/tank/jetpack/void(src)
 	new /obj/item/clothing/mask/breath(src)
-	new /obj/item/clothing/head/helmet/space/nasavoid(src)
-	new /obj/item/clothing/suit/space/nasavoid(src)
+	new /obj/effect/nasavoidsuitspawner(src)

```

Comment:
make sure this winds up in the closet and not on top of it.

Reply:
This has been tested many times, it spawns inside, even when spawning new closets.